lib/ansible/runner.rb

```diff
@@ -377,6 +378,13 @@ def python3_modules_path
       def determine_existing_python_paths_for(*paths)
         paths.select { |path| File.exist?(path) }
       end
+
+      def wait_on(dir)
+        100.times do
+          Dir.exist?(dir)
```

Comment:
I think you wanted `break if Dir.exist?` ?

Reply:
you are correct... how did I miss that...

Anyway, thanks, will change in the new PR.